torch/nn/modules/container.py

```diff
@@ -480,6 +480,13 @@ def extra_repr(self) -> str:
     def __call__(self, input):
         raise RuntimeError('ParameterList should not be called.')
 
+    def _replicate_for_data_parallel(self):
+        warnings.warn("nn.ParameterList is being used with DataParallel but this is not "
```

Comment:
"reparameterization with nn.ParameterList is not supported. Since DataParallel is a particular case of reparameterization, use of nn.ParameterList with DataParallel is not supported.